source/fundamentals/connection/connection-options.txt

```diff
@@ -192,10 +192,15 @@ parameters of the connection URI to specify the behavior of the client.
 
    * - **authMechanism**
      - string
-     - Specifies the :doc:`authentication mechanism
-       </fundamentals/auth>` that the driver should use if a credential
+     - Specifies the authentication mechanism
+       that the driver should use if a credential
        was supplied.
 
+.. TODO:(DOCSP-29261) refactor above to:
+..  Specifies the :doc:`authentication mechanism
+..  </fundamentals/auth>` that the driver should use if a credential
+..  was supplied.
```

Comment:
Question:
I'm not sure I understand what the work is in the referenced ticket or in this comment. Could someone clarify in the ticket? And once the ticket is updated, is this comment still necessary?

Reply:
ticket/comment refer to creating to the page `/fundamentals/auth` and then also linking to it here.